src/Ozone_Monitor/data_xtrct/ush/OznMon_MkBase.sh

```diff
@@ -229,18 +173,28 @@ for type in ${SATYPE}; do
             have_ctl=1
          fi
       fi
-      adate=`$NDATE +${CYCLE_INTERVAL} $cdate`
-      cdate=$adate
+
+      cdate=$($NDATE +${CYCLE_INTERVAL} $cdate)
    done
 
-   ${UNCOMPRESS} *.${Z}
+   if compgen -G "*.gz" > /dev/null; then
```

Comment:
Is .gz the only option now?  Other locations still refer to `${Z}`.  Or is this  specific to the ctl files?

Reply:
I reverted it back to ${Z}.